### DISCUSSION TITLE
npm command is not found on my machine?

### DISCUSSION THREAD

**Discussion by Archimedes2203** [community/community#75603] [Programming Help] [answered]
### Body

I'm constantly met with this error:

`sudo: npm: command not found`

Which is super annoying. WHy does it not work? THis used to work a couple of days ago.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by arienshibani** [comment, marked as answer]
If your npm file is not found in `/usr/local/bin/npm`, then simply install node.js again with the package on their website.
`sudo apt-get install npm` might also solve your issues.

**Reply by Archimedes2203** [reply]
Thank you! This worked 👍

**Reply by adeelahmadk** [reply]
The apt repository may not have the version you want to install. In this scenario it would be much simpler to symlink the `npm` and `node` binaries in `/usr/local/bin`:

```
sudo ln -s $(realpath `which npm`) /usr/local/bin/npm
sudo ln -s `which node` /usr/local/bin/node
```

**Reply by miladxandi** [reply]
> The apt repository may not have the version you want to install. In this scenario it would be much simpler to symlink the `npm` and `node` binaries in `/usr/local/bin`:
> 
> ```
> sudo ln -s $(realpath `which npm`) /usr/local/bin/npm
> sudo ln -s `which node` /usr/local/bin/node
> ```

Thanks, that worked for me also.
